CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.27.0
+
+* Add support for nested maps in `map.get`.
```

Comment:
Can you add a bit more detail here? For some percentage of users, this will be their first encounter with this feature, so it would be nice to give them at least an example of how it works, as well as maybe a link to further documentation on the website.

Reply:
Done.